src/platyPS/platyPS.psm1

```diff
@@ -950,7 +952,8 @@ function New-ExternalHelpCab
         #Creates an entry in the cab directive file for each file in the source directory (uses FullName to get fuly qualified file path and name)     
         foreach($file in Get-ChildItem -Path $CabFilesFolder -File)
         {
-            Add-Content $DirectiveFile ("'" + ($file).FullName +"'" )
+            Add-Content $DirectiveFile ("'" + ($file).FullName +"'" )            
+            Compress-Archive -DestinationPath $zipPath -Path $file.FullName -Update
```

Comment:
@jongeller do you think we should always create 2 of them, or it's better to make it an option?